to_copy/usr/bin/pep-filter.sh

```diff
@@ -141,13 +136,14 @@ prune_list() {
 
     # delete any domain lists
     docker exec encryptme bash -c "[ -f '$domain_file' ]" && {
-       docker exec -i encryptme rm -f "$domain_file"
+       docker exec -i encryptme rm -f "$list_name"
```

Comment:
Here we must delete a file (porn.blacklist) and providing a filename for that not a list name (porn)